### PR TITLE
Add support for CODEOWNERS

### PR DESCRIPTION
**Pull request by gene1wood**
Relates to #1220

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# The threat management team will be the default owners for everything in the
+# repo. Unless a later match takes precedence, these users will be requested for
+# review when someone opens a pull request.
+*                @pwnbus @mpurzynski @Phrozyn @tristanweir
```
Comment (Phrozyn) [inline CODEOWNERS]: @Phrozyn should be @phrozyn (lowercase p)

Comment (tristanweir) [reply CODEOWNERS]: image

It renders for everyone else with a capital 'P'

Comment (gene1wood) [reply CODEOWNERS]: Ya, your GitHub username shows as [`Phrozyn`](https://github.com/phrozyn) not `phrozyn`

Comment (Phrozyn) [reply CODEOWNERS]: You're absolutely right, disregard and carry on :)

**Pull request merged**